cmd/pebble/main.go

```diff
@@ -29,6 +29,17 @@ type config struct {
 	}
 }
 
+func setConfigDefaults() (config) {
+	var conf config
+	conf.Pebble.ListenAddress = "0.0.0.0:14000"
+	conf.Pebble.ManagementListenAddress = "0.0.0.0:15000"
+	conf.Pebble.Certificate = "test/certs/localhost/cert.pem"
+	conf.Pebble.PrivateKey = "test/certs/localhost/key.pem"
+	conf.Pebble.HTTPPort = 5002
+	conf.Pebble.TLSPort = 5001
+	return conf
```

Comment:
There are two values (`ocspResponderUrl` and `externalAccountBindingRequired`) which don't get initialized here. Sure, their defaults are also their zero-values, but being explicit is good.